cli/cmd/vuln_host.go

```diff
@@ -595,18 +532,14 @@ func hostVulnPackagesTable(cves []api.HostVulnCVE, withHosts bool) ([][]string,
 	return out, ""
 }
 
-func hostVulnCVEsTable(cves []api.HostVulnCVE) ([][]string, string) {
-	var out [][]string
+func hostVulnCVEs(cves []api.HostVulnCVE) ([]api.HostVulnCVE, string) {
```

Comment:
if this function is filtering please add that to the name of the function just like you did for `filterHostVulnCVEs()`